deepcell_tracking/isbi_utils.py

```diff
@@ -40,32 +40,33 @@
 from deepcell_tracking.utils import load_trks
 
 
-def trk_to_isbi(track, path):
```

Comment:
Let's try to be backwards compatible. leave the `path` argument in and default it to `None`.

If the user passes it in, warn it is deprecated (leave it in the docs as deprecated as well).

We should probably write the file at the end as well if the user is expecting it.



Please leave this as a value and default it to none.

Then in the body, use a deprecation warning.

Reply:
Leaving everything as backward compatible is very important. We may need to use external .txt files to benchmark on.

Reply:
Makes sense! A couple of questions: should we keep the `path` arg in `txt_to_graph` so that there are 3 args? And do we still need to write the file in `trk_to_isbi` if we are giving the warning?

Reply:
What I'd like (if possible) is to `trk_to_isbi` in `txt_to_graph`, and then write the file.

Maybe no deprecation value is needed here as long as the old function behaves normally. (we can warn in the `benchmark` function instead maybe).

Reply:
So `txt_to_graph` should take in `track`, `path=None`, and `node_key=None` and call `trk_to_isbi` so that only `txt_to_graph` is called in `benchmark`? 

And what is the output we are writing? Is it the previous implementation of `trk_to_isbi`, just an ISBI-formatted txt file?

Reply:
I think I'm getting confused on the names of the functions. What I'd like is for the old functions to continue to exist so old code does not break, but internally, those old functions should rely on your new code (to prevent code duplication).

We should definitely set `path=None` in the `benchmark` function. I'm now unsure about the other old functions. They may not be called by the new pattern unless the user does so explicity.

Reply:
Okay, I made some changes such that if the user did all the old calls with `path` for any function it would continue to work as the old code did, but if the user did not use paths then the new code would be used. Is this what you were hoping for?